src/ln/router.rs

```diff
@@ -215,7 +224,7 @@ pub struct RouteHint {
 /// Tracks a view of the network, receiving updates from peers and generating Routes to
 /// payment destinations.
 pub struct Router {
-	secp_ctx: Secp256k1<secp256k1::VerifyOnly>,
+	secp_ctx: Secp256k1<secp256k1::All>,
```

Comment:
Is this stale? I see no new uses of secp_ctx.

Reply:
old use, was used when I still look at storing all the individual fields.